### PR TITLE
Exclude the EPTI attribute on the non SAML forms

### PR DESCRIPTION
**Pull request by MKodde**
This was a left-over from the #531 PR. Where the Oidc form would display the EPTI attribute. Something that should not be the case.

### CODE REVIEW

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Dto/AttributeForm.php [hunk 1]
```diff
@@ -20,22 +20,28 @@
...
     }
 
-    public static function fromForm(array $languages): ?AttributeForm
+    public static function fromForm(array $form): ?AttributeForm
     {
-        return new self(
-            $languages
-        );
+        $excludeFrom = [];
+        if (array_key_exists('excludeOnEntityType', $form)) {
```
Comment (Shaky212) [inline src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Dto/AttributeForm.php:42]: Is it justified to use a constant for 'excludeOnEntityType'?

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/OidcngEntityType.php [hunk 2]
```diff
@@ -366,7 +366,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
     private function buildAttributeTypes(FormBuilderInterface $container): FormBuilderInterface
     {
-        return $this->attributeTypeFactory->build($container);
+        return $this->attributeTypeFactory->build($container, Constants::TYPE_OPENID_CONNECT_TNG);
```
Comment (Shaky212) [inline src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/OidcngEntityType.php:369]: Not for now, but maybe an idea for the future to divide a large builder into smaller methods to apply structure,  methods like buildMetadata, buildOrganization, buildAttributesTypes, etc

File: app/config/attributes.json [hunk 3]
```diff
@@ -2,9 +2,11 @@
   {
     "id": "givenName",
     "form" : {
-      "en": {
-        "label": "Given name attribute",
-        "info": "Text should be set in web translations"
+      "translations" : {
```
Comment (Shaky212) [inline app/config/attributes.json:5]: Adding translations makes sense and gives more structure!

**Review by Shaky212 — APPROVED** [review]
Nice solution to exclude an attribute from the premises. Added some suggestions/remarks but no show stoppers.

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Dto/AttributeForm.php (hunk 1 above)
Comment (MKodde) [reply src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Dto/AttributeForm.php:42]: Good suggestion. At this point, this is the only place where we consume that key in the application. And it should remain the only place. I do not see the need to use a class constant in this case.

**Comment by MKodde** [comment]
Merging :shipit:

**Pull request merged**